data_model/src/events/mod.rs

```diff
@@ -133,7 +155,7 @@ pub trait Filter {
 }
 
 #[cfg(feature = "transparent_api")]
-impl Filter for FilterBox {
+impl EventFilter for EventFilterBox {
```

Comment:
this here is a perfect example of the reason why I renamed `Event` to `EventBox`

Reply:
Ah, yeah, fair